### DISCUSSION TITLE
Difference between Set-PoshPrompt & copy raw template

### DISCUSSION THREAD

**Discussion by tomkerkhove** [JanDeDobbeleer/oh-my-posh#1157] [Troubleshoot] [answered]
I've had some issues when copying the raw template to  `$Home\.oh-my-posh.omp.json` but when using `Set-PoshPrompt` it works fine.

Does it do any special things? Was a bit confusing but luckily the docs helped.

**Comment by JanDeDobbeleer** [comment, marked as answer]
What do you mean with special things? I don't think I'm following along :-) Is that the reason you made the changes to the doc?

**Reply by tomkerkhove** [reply]
Nevermind, think it's PEBKAC so resolving :)
